blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/EnumValidator.java

```diff
@@ -70,8 +68,8 @@ private void validateThresholdAndWrite(Locator locator, List<String> currentEnum
             // count of current enum values of metric exceeded threshold, bad metric
             // write locator to bad metric table
             try {
-                getWriter().writeExcessEnumMetric(locator);
-            } catch (ConnectionException e) {
+                IOContainer.fromConfig().getExcessEnumIO().insertExcessEnumMetric(locator);
+            } catch (Exception e) {
                 log.error(String.format("Exception writing bad metric %s: %s", locator.toString(), e.getMessage()), e);
```

Comment:
You don't need to do ```e.getMessage()``` here. When the exception is passed in as a 2nd parameter to ```log.error()```, the log4j will print the exception message.